tests/staggered_dslash_ctest.cpp

```diff
@@ -3,17 +3,18 @@
 using namespace quda;
 
 bool ctest_all_partitions = false;
+bool ctest_domain_decomposition = false;
 
 using ::testing::Bool;
 using ::testing::Combine;
 using ::testing::Range;
 using ::testing::TestWithParam;
 using ::testing::Values;
 
-class StaggeredDslashTest : public ::testing::TestWithParam<::testing::tuple<int, int, int>>
+class StaggeredDslashTest : public ::testing::TestWithParam<::testing::tuple<int, int, int, int, int>>
```

Comment:
The new enums `QudaDomainDecompositionType` and `QudaDomainDecompositionColor` also need to be propagated to the staggered tests